.travis.yml

```diff
@@ -9,13 +9,13 @@ os:
   - linux
 
 script:
-  - chmod 777 .
+  - chmod -R 777 .
   - docker run -it --cap-add SYS_ADMIN --device /dev/fuse -v $PWD:/stratus -w /stratus a12e/docker-qt:5.12-gcc_64 sh build-in-docker.sh
 
 after_success:
   - wget -c https://github.com/probonopd/uploadtool/raw/master/upload.sh
-  - if [ "$TRAVIS_BRANCH" != "$TRAVIS_TAG" ] && [ "$TRAVIS_BRANCH" != "master" ]; then export TRAVIS_EVENT_TYPE=pull_request; fi
-  - bash upload.sh Stratus-*.AppImage
+  # only upload the Docker image for CI builds triggered by tags
```

Comment:
hm, we want the master build to be uploaded though

Reply:
here: https://github.com/status-im/nim-stratus/releases/tag/continuous

Reply:
Even after seeing the Travis build being blocked (and killed after 50 minutes) by an upload speed of 0.45 KB/s?

Even if you're willing to risk that, TRAVIS_BRANCH points to "master" for al PRs that target the master branch, so you end up with PR builds in your "continuous" tag.

Reply:
the slow uploads happen because when the thing is tagged `pull_request` it'll be uploaded to some other service: https://github.com/probonopd/uploadtool/blob/master/upload.sh#L104 - I presume this service used to work better, but I think the idea here is that it's possible to download the built PR binary from _somewhere_. it's a nice thought but we can live without it so it's fine to disable completely instead of calling `upload.sh`.

the intent is that pr builds are not uploaded to continuous release, but that master is, and that any tagged released (git tag) get their own release separate from continuous.

Reply:
I'm fine with merging this branch and fixing master in case it doesn't get published to the continuous tag.. I'm not sure how the environment variables work, I think I just copy-pasted from some random place.

Reply:
editorial note: there's release tags and git tags involved. the former, we can call releases, such as the `continuous` release, to avoid confusion.

Reply:
So we copy and modify the upload script in the repo?

Reply:
well, or we simply don't call it from PR's

Reply:
Found the relevant variable for that: TRAVIS_PULL_REQUEST is "false" outside PRs.

Reply:
> there's release tags and git tags involved. the former, we can call releases, such as the `continuous` release, to avoid confusion.

It's the same thing. Run `git pull --tags --force; git tag` in your local clone and you'll see a "continuous" tag. It also appears in `git log`.